### PR TITLE
[Cleanup] Bump Node version, migrate to Eslint, start cleaning up lint errors

### PR DESCRIPTION
**Pull request by kevinjcliao**
Hi, 
I'm a nobody in g0v and a student of Taigi. I'm using itaigi a lot and would like to give back. 

This PR is part of a series of changes I'd like to contribute to clean up the itaigi codebase. 

This PR: 
1. Bumps node version from the current version (5) to the latest version (15). 
2. Adds eslint as a dependency
3. Starts cleaning up the code as per eslint.

### CODE REVIEW

File: src/GuanKiann/LokIm/ClientRecorder.js [hunk 1]
```diff
@@ -1,174 +1,173 @@
+const MainWorkerScript = require('raw!./RecordWorker.js');
+var EncoderWorkerScript = require('raw!./mp3Worker.js');
```
Comment (chunyenHuang) [inline src/GuanKiann/LokIm/ClientRecorder.js:2]: Still using `var` here?

Comment (kevinjcliao) [reply src/GuanKiann/LokIm/ClientRecorder.js:2]: good find! I was using eslint to automatically fix things. There are still 190,000 errors in eslint, but we're down from 390,000 errors so this is progress. :P

**Comment by kevinjcliao** [comment]
The Travis CI integration is broken because it's still using the old YML file, which relies on Node version 6, and not the Node version 15, which I'm using here.

**Comment by sih4sing5hong5** [comment]
> The Travis CI integration is broken because it's still using the old YML file, which relies on Node version 6, and not the Node version 15, which I'm using here.

若是bô希望語法檢查影響CI，就毋免走`- TSI2_LING7='test'`

https://github.com/g0v/itaigi/blob/88a174fb668d66dd655b0664da25bbda5b51dd03/.travis.yml#L7

**Review by sih4sing5hong5 — APPROVED** [review]
感謝tshiâu排版 :)

**Pull request merged**